packages/tidy5/tidy5.0.1.1/opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "1.2"
+maintainer: "zandoye@gmail.com"
+authors: [ "ZAN DoYe" ]
+homepage: "https://bitbucket.org/zandoye/tidy5/"
+bug-reports: "https://bitbucket.org/zandoye/tidy5/issues"
+license: "MIT"
+tags: [
+  "html"
+  "parse"
+  "tidy"
+]
+build: [
+  ["make"]
```

Comment:
Could you please change `"make"` to `make` (here and in `install`) so that OPAM will use the appropriate command on each platform?